fuzzers/005-tilegrid/generate.py

```diff
@@ -491,6 +547,20 @@ def db_add_bits(database, segments):
                     ("DSP", "CLB_IO_CLK"): (28, 2, 10),
                     ("INT_INTERFACE", "CLB_IO_CLK"): (28, 2, None),
                     ("BRAM_INT_INTERFACE", "CLB_IO_CLK"): (28, 2, None),
+                    # TODO: These are all guesses copied from CLBLL.
```

Comment:
you should be able to use the scraper tool to confirm the number of frames

Reply:
Scraper tool indicates this might not be right (IOB_X0Y0 width: 42):

```
$ cat build/design_IOB_X0Y0.delta
+bit_00400027_000_29
$ $XRAY_BLOCKWIDTH build/design_IOB_X0Y0.bit |grep 0x00400000
0x00400000: 0x2A
```

But it might be just as well to commit for now and we'll sort it out later